Dockerfile

```diff
@@ -26,58 +26,49 @@ ARG RVM_VERSION=stable
 ENV RVM_VERSION=${RVM_VERSION}
 
 # RMV user to create
+# Optional: child images can change to this user, or add 'rvm' group to other user
 ARG RVM_USER=rvm
 ENV RVM_USER=${RVM_USER}
 
-# Install RVM
-RUN apt update \
-    && apt install -y \
-       curl \
-       git \
-       gnupg2 \
+# Install RVM dependencies
+RUN sed -i 's~http://archive\(\.ubuntu\.com\)/ubuntu/~mirror://mirrors\1/mirrors.txt~g' /etc/apt/sources.list \
+    && apt-get update -qq \
+    && apt-get install -qy --no-install-recommends \
+       ca-certificates curl netbase \
+       git openssh-client \
```

Comment:
What requires `openssh-client`?

Reply:
Because of `--no-install-recommends` switch, it will install `git` without some packages such as `openssh-client`, `less`, `kbr5`.
So, we need to decide what packages need to install here, e.g. `ca-certificates`, `openssh-client` (maybe not needle)